.github/workflows/version.yaml

```diff
@@ -40,16 +40,26 @@ jobs:
       - name: NEWS.md and DESCRIPTION Version check 🏁
         run: |
           DESC_VERSION=$(awk -F: '/Version:/{gsub(/[ ]+/,"") ; print $2}' DESCRIPTION)
-          NEWS_VERSION=$(awk '/^#+ /{print $3; exit}' NEWS.md)
+          NEWS_VERSION=$(awk '/^#+ /{print $3,$4; exit}' NEWS.md)
+          DESC_DEV_VERSION=$(echo $DESC_VERSION | awk -F '.' '{print $NF}')
           echo "NEWS.md version: $NEWS_VERSION"
           echo "DESCRIPTION version: $DESC_VERSION"
-          if (test $DESC_VERSION = $NEWS_VERSION ); then
-              echo "NEWS.md and DESCRIPTION have the same version"
-          else
-              echo "🙈 NEWS.md and DESCRIPTION have different versions!"
-              echo "🙏 Please fix this."
-              exit 1
+          if [[ "$DESC_VERSION" == "$NEWS_VERSION" ]]
+          then {
+            echo "NEWS.md and DESCRIPTION have the same version 🎉"
+            exit 0
+          }
           fi
+          if [[ $DESC_DEV_VERSION -gt 9000 && "$NEWS_VERSION" == "(development version)" ]]
```

Comment:
Not sure but if `9000` is a valid development version (right after release), then maybe `-gt` should be changed to `-ge`?